Components/Badges/badges.html

```diff
@@ -72,6 +72,7 @@
         </nav>
         <div class="aside-wrapper">
           <h1 class="heading">BADGES</h1>
+          <h2 class="text-grey">Different type of badges used for notification</h2>
           <div class="flex-col copy">

```

Comment:
can you explain the "copy" class?

Reply:
copy class is to copy the code snippet Javascript is working